src/PolyhedralGeometry/PolyhedralComplex/constructors.jl

```diff
@@ -0,0 +1,86 @@
+###############################################################################
+###############################################################################
+### Definition and constructors
+###############################################################################
+###############################################################################
+
+struct PolyhedralComplex
+    pm_complex::Polymake.BigObject
+    function PolyhedralComplex(pm::Polymake.BigObject)
+        return new(pm)
+    end
+end
+
+pm_object(pc::PolyhedralComplex) = pc.pm_complex
+
+
+@doc Markdown.doc"""
+    PolyhedralComplex(Polytopes, VR, far_vertices, Lineality)
+
+# Arguments
+- `Polytopes::IncidenceMatrix`: An incidence matrix; there is a 1 at position
```

Comment:
Argument names should be lower case, unless the argument is a type (but then you usually would not do `SomeTypeName::Type` but rather `::Type{SomeTypeName}`).

Same applies to `VR`, too